src/lib/db.js

```diff
@@ -1350,4 +1412,39 @@ export default class db {
       })
     });
   }
+  
+  addTeam(teamObject) {
+    return new Promise((resolve) => {
+      const insertObject = {};
+      _.forEach(teamObject, (value, key) => {
+        insertObject[mapGhostNames(key)] = value;
```

Comment:
Hmm.... I'm a bit in doubt whether I want the mapGhostNames here or not. It's of no use as id, slug, name and description are all simple and good names, so it's redundant. Overall it's a quite ugly function to have to use, and also since one of the things it maps is not from Ghost anymore we should also change the name of it.

As of now I think just put in key instead of mapGhostNames(key).